worlds/dark_souls_3/Items.py

```diff
@@ -1,36 +1,1276 @@
-import sys
+from enum import Enum
+from typing import NamedTuple
 
 from BaseClasses import Item
-from worlds.dark_souls_3.data.items_data import item_tables, dlc_shields_table, dlc_weapons_upgrade_10_table, \
-    dlc_weapons_upgrade_5_table, dlc_goods_table, dlc_spells_table, dlc_armor_table, dlc_ring_table, dlc_misc_table, dlc_goods_2_table
 
 
-class DarkSouls3Item(Item):
-    game: str = "Dark Souls III"
+class DS3ItemCategory(Enum):
```

Comment:
I would personally go for IntEnum here (and for LocationCategory), but it's up to you. Either will work.

Reply:
Changed in cd371d2

Can I ask what the purpose is? I suppose was working under the assumption that `IntEnum` was for when you needed your values to also be ints, and `Enum` was sort of the "default".

Reply:
That assumption is correct, however IntEnums are guaranteed to only have integer values, which is cleaner imo if you go with integer values (i.e. you don't use `auto()`). Unless you serialize or de-serialize it, or have special pseudo-flags, they behave the same.